src/main/java/io/reactivex/Single.java

```diff
@@ -2649,6 +2649,26 @@ public final T blockingGet() {
         return toSingle(toFlowable().retry(predicate));
     }
 
+    /**
+     * Repeatedly re-subscribe at most times or until the predicate returns false, whichever happens first
```

Comment:
Let's make it more consistent with javadoc you've added for Completable? It didn't mention `Repeatedly` and was emphasizing error condition, while with this javadoc you only realize in the end of it that error is initial trigger for whole operator

Reply:
It should be yeah but if we only change this method then this one is inconsistent with all of the other Single methods.